.github/workflows/publish.yml

```
@@ -0,0 +1,44 @@
+name: Deploy Script and sample to Github Pages
+
+on:
+  release: 
```

yamllint

[error] trailing spaces

(trailing-spaces)